docs/en/operation/Journal.md

```diff
@@ -104,11 +104,9 @@ If this is not set, clients will look for masters using the hostnames from `allu
 and the master rpc port (Default:`19998`).
 
 ### Advanced configuration
+
 * `alluxio.master.embedded.journal.transport.request.timeout.ms`: The duration after which embedded journal masters will timeout messages sent between each other.
  Lower values might cause leadership instability when network is slow. Default: `5s`.
-* `alluxio.master.embedded.journal.appender.batch.size`: Size of a single internal knowledge batch sent from leader to secondary masters.
```

Comment:
@bf8086 are the following properties need to be added as advanced configuration:
```
MASTER_EMBEDDED_JOURNAL_CATCHUP_RETRY_WAIT
MASTER_EMBEDDED_JOURNAL_WRITE_LOCAL_FIRST_ENABLED 
MASTER_EMBEDDED_JOURNAL_SNAPSHOT_REPLICATION_CHUNK_SIZE
```

Reply:
Yes.